js/index.js

```diff
@@ -0,0 +1,325 @@
+//Function to validate form
+function validateForm() {
+    var productid = document.getElementById("productid").value;
+    var productname = document.getElementById("productname").value;
+    var price = document.getElementById("price").value;
+    var description = document.getElementById("description").value;
+    var regex = /^[^\s][a-zA-Z]+[a-zA-Z0-9 ]*[^\s]$/;
+
+    if (productid == "" || productname == "" || price == "" || description == "") {
+        alert("All fields are required!");
+        return false;
+    }
+    else if (isNaN(productid)) {
+        alert("Only numeric Value");
+        return false;
+    }
+    else if (price < 0) {
+        alert("Price must be positive.")
```

Comment:
why you allowing negative price. 
Is that possible logically? Did you seen anywhere any e-commerce site add negative product price? If yes so how you can pay it?

Reply:
sir in this case if any person tries to add negative integers will shoes an error as price must be positive. else it can take - values.